supporting_content/columns/regionname.md

```diff
@@ -0,0 +1,53 @@
+# Column: Region Name
+	
+## Example provider mappings
```

Comment:
Lets not copy / paste the RegionId here. Instead in other places we've linked to that other field using a ../regionid.md type notation. Feel free to capture anything new here though.